lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": ["packages/*"],
+  "version": "14.0.1",
```

Comment:
```suggestion
  "version": "14.2.1",
```

Reply:
I think this number will be bumped to 15, but im not sure :s